x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -381,6 +415,30 @@ export class AlertsClient {
     }
   }
 
+  /**
+   * When an update by ids is requested, do a multi-get, ensure authz and audit alerts, then execute bulk update
+   * @param param0
+   * @returns
+   */
+  private async mgetAlertsAuditOperateStatus({
+    ids,
+    status,
+    indexName,
+    operation,
+  }: {
+    ids: string[];
+    status: STATUS_VALUES;
+    indexName: string;
+    operation: ReadOperations.Find | ReadOperations.Get | WriteOperations.Update;
+  }) {
+    return this.mgetAlertsAuditOperate({
```

Comment:
I make `mgetAlertsAuditOperate` generic to accept any field and I created the `mgetAlertsAuditOperateStatus` for the status